src/frontend/templates/components/Footer.js

```diff
@@ -40,7 +40,11 @@ export const Footer = () => {
         :
         (<Link to='/login'><span className='f-w-800 txt-over'>Wishlist </span></Link>)
         }
-        <span><span className='f-w-800 txt-over'>Cart</span></span>
+         { userState.foundUser ?
+        (<Link to='/cart'><span className='f-w-800 txt-over'>Cart</span></Link>)
+        :
+        (<Link to='/login'><span className='f-w-800 txt-over'>Cart </span></Link>)
+        }
```

Comment:
Instead of two different Link elements here, you can just add a ternary operator to the route of the Link.

Reply:
> Instead of two different Link elements here, you can just add a ternary operator to the route of the Link.
hi tejas thanks for the review , will work upon it !